pkg/env/armhelper.go

```diff
@@ -83,21 +81,21 @@ func newARMHelper(ctx context.Context, log *logrus.Entry, env Interface) (ARMHel
 	}
 
 	options := env.Environment().ClientCertificateCredentialOptions()
-	tokenCredential, err = azidentity.NewClientCertificateCredential(env.TenantID(), os.Getenv("AZURE_ARM_CLIENT_ID"), certs, key, options)
+	spTokenCredential, err := azidentity.NewClientCertificateCredential(env.TenantID(), os.Getenv("AZURE_ARM_CLIENT_ID"), certs, key, options)
```

Comment:
can we rename this `armHelperTokenCredential`? Or something like that.

Reply:
Pushed a fixup commit for this.  I also renamed the authorizer to keep it consistent.